lib/routemap_cli.c

```diff
@@ -163,10 +185,38 @@ void route_map_instance_show(struct vty *vty, struct lyd_node *dnode,
 	/* Print route map `set` for old CLI users. */
 	for (rmr = rmi->set_list.head; rmr; rmr = rmr->next) {
 		/* Skip all sets implemented by northbound. */
+		SKIP_RULE("ip next-hop");
```

Comment:
If we have all route maps migrated, then this function is no longer necessary.

Reply:
Addressed